src/react-apps/applications/runtime/src/shared/components/altinnParty.tsx

```diff
@@ -90,7 +90,7 @@ const styles = createStyles({
   },
   subUnitTextWrapper: {
     borderTop: `1px solid ${altinnTheme.altinnPalette.primary.greyMedium}`,
-    padding: 21,
+    paddingRight: 21,
```

Comment:
Should this have REMs instead of px?

Reply:
Does it make any difference? My guess is that if we should use REMs, we should use it throughout the whole application? And that's probably best handled in an issue of its own?

Reply:
Let it be.